### ISSUE TITLE
Create slot req body has some issues

### ISSUE THREAD

**Issue by Destravna** [dyte-submissions/vit-hiring-2023-phase1-test#8]
Request with just an array of string in timings was made by the python program whereas the one with right timing values is made using postman
<img width="958" alt="image" src="https://user-images.githubusercontent.com/71959463/222915340-ba49dc41-128c-408a-8fa1-d2dd5f0f3286.png">

The timings array in body is received as an array of strings at server, I tried to mimic the request using postman, for that it was fine.

<img width="750" alt="image" src="https://user-images.githubusercontent.com/71959463/222915456-e2d296f2-1859-47b8-b0ab-c56a6294c460.png">

**Comment by Tanmay000009** [comment]
Facing a similar issue. 

It needs a two-stop solution.

- setting header: `'Content-Type': 'application/json'`
- converting body in python to json string using json.dumps
```
body_json = json.dumps(body)
r = request_function(url(path), body_json, headers=headers)
```

**Comment by Destravna** [comment]
yeah and need to remove content type from headers to make sure courses are added.
